configure.ac

```diff
@@ -363,17 +363,16 @@ else
 fi
 AM_CONDITIONAL(HAVE_LIBEXIF, test x$libexif_pkgconfig = xyes)
 
-dnl
```

Comment:
Wouldn't it be better to keep the X11 support by default, and only remove it on Mac?